spec/migrations/20200331150436_rename_foreman_features_spec.rb

```diff
@@ -0,0 +1,58 @@
+require_migration
+
+describe RenameForemanFeatures do
+  class MiqRolesFeature < ActiveRecord::Base; end
+
+  let(:user_role_id) { rand(1..1000) }
```

Comment:
To deal with regions accurately, prefer something like

`ActiveRecord::IdRegions::Migration.anonymous_class_with_id_regions.rails_sequence_start`

Reply:
If you really want a random id just add `+ rand(1...1000)` to the end of that

Reply:
@bdunne We could really use a cleaner helper method for stuff like this... maybe a spec/support thing in activerecord-id_regions gem?  I'm thinking a handful of helpers for the most common stuff.

Reply:
Yeah, that would make it much easier to use